src/store/initial-state.test.js

```diff
@@ -29,6 +29,19 @@ describe('mergeLocalStorage', () => {
     window.localStorage.clear();
   });
 
+  it('overrides selected tag values in state with localstorage values if provided', () => {
+    const localStorageValues = {
+      tag: { enabled: 'large' }
+    };
+    saveState(localStorageValues);
+    expect(
+      mergeLocalStorage({
+        tag: { enabled: 'large' }
+      })
+    ).toMatchObject(localStorageValues);
+    window.localStorage.clear();
+  });
+
```

Comment:
I might be wrong but this test seems redundant to me - it's just a modified version of the previous test. Is it necessary to check that this function specifically merges tag values, if we've already established that it already merges other values? The only difference that I can see is that this value is a nested object, in which case it should just be added to the previous test's input and expected result.

Reply:
I agree, I actually had the same thoughts when I added this test, but in the end decided to add it to partly demonstrate / document the type of data we are storing on the localStorage for tags, yet the intention behind this test is the same as the previous test.

In this case I will combine this with the previous one by adding this extra type of data in, and deleting this test.